packages/gafl-webapp-service/src/pages/contact/contact/__tests__/result-function.spec.js

```diff
@@ -0,0 +1,32 @@
+import resultFunction from '../result-function'
+import { CommonResults } from '../../../../constants.js'
+
+describe('contact > result-function', () => {
+  const mockStatusCacheGet = jest.fn()
+
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        status: {
+          getCurrentPermission: mockStatusCacheGet
+        }
+      }
+    })
+  }
+
+  describe('default', () => {
+    beforeEach(jest.clearAllMocks)
+
+    it('should return ok if isLicenceForYou is true', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: true }))
+      const result = await resultFunction(mockRequest)
+      expect(result).toBe(CommonResults.OK)
+    })
+
+    it('should return summary if isLicenceForYou is false', async () => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: false }))
+      const result = await resultFunction(mockRequest)
+      expect(result).toBe(CommonResults.SUMMARY)
+    })
+  })
+})
```

Comment:
Please will you extend these tests to cover the other parts of the expression (`status.fromSummary` === `CONTACT_SUMMARY_SEEN` and `status.renewal`)